File: chapter_5/challenge1.c
// Challenge 1, Chapter 5
// jeffreydiaz27@gmail.com

#include <stdio.h>

float remainingAngle(float angleA, float angleB)
{
	return 180 - (angleA + angleB);
}

int main(int argc, const char* argv[])
{
	float angleA = 30.0;
	float angleB = 60.0;
	float angleC = remainingAngle(angleA, angleB);
	printf("The third angle is %.2f\n", angleC);
	return 0;
}

File: chapter_13/challenge1.m
// Challenge 1, Chapter 13
// jeffreydiaz27@gmail.com

#import <Foundation/Foundation.h>

int main(int argc, const char* argv[])
{

	@autoreleasepool {
		NSHost*   my_host   = [NSHost currentHost];
		NSString* host_name = [my_host localizedName];
		
		NSLog(@"Host name: %@\n", host_name);
	}
    return 0;
}
